src/index.js

```diff
@@ -468,14 +482,18 @@ class Masq {
     if (!this.dataEncryptionKey) throw Error('Data encryption key is not set')
   }
 
+  _checkNonce () {
+    if (!this.nonce) throw Error('Nonce is not set')
+  }
```

Comment:
function _checkNonce and _checkDEK never called ?

Reply:
yes removed.